File: 2.0/java/contacts/getContactList.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.contacts.model.ContactList;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api login", "api password");
        ContactList contactList = client.contactListsApi().get(379506003L, "id,name,size,created");
    }
}

File: 2.0/csharp/contacts/getContactList.cs
using CallfireApiClient;
using CallfireApiClient.Api.Contacts.Model;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        ContactList contactList = client.ContactListsApi.Get(379506003, "id,name,size,created");
    }
}
